docs/tutorials/markdown.md

```diff
@@ -0,0 +1,144 @@
+# Formatting
+
+Modrinth allows the usage of [GitHub Flavourited Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for text fields such as your Project's Description or Version description.
+
+To help you understand what is and isn't available to use is this page here listing all currently supported Markdown features.
+
+## Basic formatting
+
+| Markdown                    | Result                |
+| --------------------------- | --------------------- |
+| \*\*Bold\*\*                | **Bold**              |
+| \*Italic\*                  | *Italic*              |
+| \*\*\*Bold and Italic\*\*\* | ***Bold and italic*** |
+| \~\~Strikethrough\~\~       | ~~Strikethrough~~     |
+
+## Code Blocks
```

Comment:
Headings should all be uncapitalised besides the first letter of the first word

Reply:
Changed